### DISCUSSION TITLE
skywalking homepage live demo cannot be opened

### DISCUSSION THREAD

**Discussion by kestrelcjx** [apache/skywalking#11917] [General]
<img width="1263" alt="截屏2024-02-23 10 34 49" src="https://github.com/apache/skywalking/assets/38725513/b1c678a5-4274-4f95-8f78-f1e667238165">
I tried accessing the live demo but it seems to be down

**Comment by Superskyyy** [comment]
@kestrelcjx yes it seems like indeed down. we will get to it.

**Comment by Superskyyy** [comment]
It's back thanks to @kezhenxu94

**Comment by wu-sheng** [comment]
It is back to work
